dev/com.ibm.ws.jbatch.security/resources/com/ibm/ws/jbatch/security/resources/BatchSecurityMessages.nlsprops

```diff
@@ -52,4 +52,17 @@ USER_UNAUTHORIZED_NO_BATCH_ROLES.useraction=If the user is not authorized to be
 #{0} The invocation subject on the thread.
 USER_UNAUTHORIZED_TO_START_JOB=CWWKY0304W: User {0} is not authorized to start batch jobs.
 USER_UNAUTHORIZED_TO_START_JOB.explanation=The user is not part of a batch role that is authorized to start batch jobs.
-USER_UNAUTHORIZED_TO_START_JOB.useraction=If the user is not authorized to start batch jobs, then no action is required. If the user is authorized to perform the requested batch operation, then configure the batch security authorization and add this user to the required batch roles.
\ No newline at end of file
+USER_UNAUTHORIZED_TO_START_JOB.useraction=If the user is not authorized to start batch jobs, then no action is required. If the user is authorized to perform the requested batch operation, then configure the batch security authorization and add this user to the required batch roles.
+
+#{0} The job instance id number
+#{1} The userid issuing the request
+#{2} A list of group names to which the job instance is accessible by
+USER_GROUP_UNAUTHORIZED_JOB_INSTANCE=CWWKY0305W: Access to job instance {0} denied.  The job has an operation group name defined and the user {1} has batchGroupMonitor or batchGroupAdmin authority but is not a member of the following configured groups: {2}.
```

Comment:
Access to the {0} job instance was denied.  The job has an operation group name defined and the {1} user has batchGroupMonitor or batchGroupAdmin authority; however, that user is not a member of the following configured groups: {2}.